test/src/main/java/org/apache/accumulo/test/functional/AddSplitIT.java

```diff
@@ -102,6 +105,31 @@ public void addSplitTest() throws Exception {
     }
   }
 
+  @Test
+  public void addSplitWithMergeabilityTest() throws Exception {
+
+    String tableName = getUniqueNames(1)[0];
+    try (AccumuloClient c = Accumulo.newClient().from(getClientProps()).build()) {
+      c.tableOperations().create(tableName);
+
+      TreeSet<Text> splits = new TreeSet<>();
+      splits.add(new Text(String.format("%09d", 333)));
+      splits.add(new Text(String.format("%09d", 666)));
+
```

Comment:
Would be good to add a bit more variety to this test.   Can test maps where the values differ and can also test the three different types of TabletMergeability.  Maybe that all could be done in one call to put splits.